package.json

```diff
@@ -1,15 +1,36 @@
 {
-  "name": "code-exercise",
+  "name": "tutorial",
   "version": "1.0.0",
-  "description": "This is a shopping cart coding exercise",
+  "description": "This is a Readme file.",
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "canned -p 5000 ./server"
+    "build": "webpack",
+    "dev": "webpack-dev-server --config webpack.dev.config.js --hot",
+    "start": "webpack-dev-server"
   },
+  "keywords": [],
   "author": "",
-  "license": "MIT",
-  "dependencies": {
-    "canned": "^0.3.12"
+  "license": "ISC",
+  "devDependencies": {
```

Comment:
figure about between devDependencies and dependencies, as many are not devDependency for your project.